src/App.tsx

```diff
@@ -10,31 +14,29 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setselectedTabId] = useState(tabs[0]);
```

Comment:
It's not id here, it's a tab
```suggestion
  const [selectedTab, setselectedTab] = useState(tabs[0]);
```